Base/usr/share/man/man7/Mitigations.md

```diff
@@ -0,0 +1,139 @@
+## Name
+
+Mitigations - Security mitigations implemented by SerenityOS.
+
+## Description
+
+Although Serenity is a hobby operating system, the developers have put substantial effort into
+adding Security mitigations to enhance the security of the system. The goal of this document is
+to collect and describe the mitigations in one centralized place.
```

Comment:
Let's not self-deprecate about being a "hobby" operating system. Of course we are, but let's take ourselves seriously :)

Reply:
Sure :+1: